### ISSUE TITLE
Improve power consumption prediction model

### ISSUE THREAD

**Issue by bdzyubak** [bdzyubak/torch-control#26]
The initial model had trouble predicting extremes. 
projects\MachineLearning\energy_use_time_series_forecasting\time_series_forecasting_energy_use.py
![image](https://github.com/bdzyubak/torch-control/assets/37943739/8b0c3215-86d6-4ef0-8191-689a23e9a481)

1) Improve prediction at extremes. 
2) Add other metrics for better human readability. 
No specific RMSE target exists at this time, so just report results.

**Comment by bdzyubak** [comment]
The original trainval performance as evaluated with 10 trees. Newer runs indicate an overfitting issue with train rmse falling to 0 with 20 trees, while the val rmse hovers around the value of 1600. Thus, the model is able to fit the data with 20 trees but has an overfitting issue. Look at regularization with the higher tree numbers, and also learning rate scheduling: 
![image](https://github.com/bdzyubak/torch-control/assets/37943739/1731f779-aef8-4b4d-80e5-0b638cd2af6e)

**Comment by bdzyubak** [comment]
Predictions are substantially improved with respect to the baseline. Closing for now until a specific RMSE (or other metric) target is defined for this project
![image](https://github.com/bdzyubak/torch-control/assets/37943739/5a3a10d2-fbec-4d7d-a239-deab7efc35a4)
